packages/jest-mock-apollo/src/Requests.ts

```diff
@@ -13,7 +13,7 @@ export default class Requests {
 
   nth(index: number) {
     return index < 0
-      ? this.requests[this.requests.length - index]
+      ? this.requests[this.requests.length - 1 + index]
```

Comment:
I think this is a bug. If index is negative, subtracting it from length would go outside the bounds of the array right?

I assumed the intention was passing -N would be Nth from the end, ex. -1 would be 2nd last